[docs]: Create a troubleshooting section (#219)
hyperledger/iroha-2-docs: src/guide/kotlin-java.md

}
```

> You **should** replace the SNAPSHOT in the configuration above with a

Reviewer: ```suggestion
You **should** replace the SNAPSHOT in the above configuration with the
```

latest `iroha-java` snapshot.

Snapshot versions match the Git commits. To get the latest snapshot, simply
visit the
[`iroha-java`](https://github.com/hyperledger/iroha-java/tree/iroha2-dev)

> repository on `iroha-2-dev` branch and copy the short hash of the last

Reviewer: ```suggestion
repository on the `iroha-2-dev` branch and copy the short hash of the last
```

commit on the main page.

![](/img/iroha_java_hash.png)

You can also check the
[commit history](https://github.com/hyperledger/iroha-java/commits/iroha2-dev)

> and copy the commit hash of one of the previous commits.

Reviewer: ```suggestion
and copy the commit hash of a previous commit.
```
